<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
  <meta name="description" content="">
  <meta name="author" content="">

  <title>Carousel Template for Bootstrap</title>

  <!-- Bootstrap core CSS -->
  <link href="../carousel/css/bootstrap.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="carousel.css" rel="stylesheet">
</head>


<!-- NAVBAR
================================================== -->
<body>
<div class="navbar-wrapper">
  <div class="container">

    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="https://2019.igem.org/Team:SYSU-CHINA">
            <img src="http://2019.igem.org/wiki/images/1/1f/T--SYSU-CHINA--title0.jpg" alt="logo"
                 style="left:30px; width: 300px; height: 50px; ">
          </a>
        </div>

        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-right .navbar-fixed-top">
            <!-- TEAM -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                TEAM
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="#">Team Members</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Collaborations">Collaborations</a></li>
                <li><a href="#">Awards</a></li>
              </ul>
            </li>

            <!-- PROJECT -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true"
                 aria-expanded="false">
                PROJECT
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Description">Description</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Design">Design</a></li>
                <li><a href="#">Experiments</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Measurement">Measurement</a></li>
                <li><a href="#">Notebook</a></li>
                <li><a href="#">Contribution</a></li>
                <li><a href="#">Results</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Demonstrate">Demonstrate</a></li>
                <li><a href="#">Improve</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Attributions">Attributions</a></li>
              </ul>
            </li>

            <!-- Model -->
            <li class="dropdown">
              <a href="http://2019.igem.org/Team:SYSU-CHINA/Model" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                MODEL
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Description">Description</a></li>
                <li><a href="#">S</a></li>
                <li><a href="#">S</a></li>
                <li><a href="#">R</a></li>
              </ul>
            </li>

            <!-- PARTS -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                PARTS
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Parts">Overview</a></li>
                <li><a href="#">Basic Parts</a></li>
                <li><a href="#">Part Collection</a></li>
              </ul>
            </li>

            <!-- SAFETY -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                SAFETY
              </a>
            </li>

            <!-- HUMAN PRACTICES -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false" style="width: 250px;">
                HUMAN PRACTICES
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Human_Practices">Human Practices</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Human_Practices#IHP">Integrated Human Practices</a></li>
              </ul>
            </li>

            <!-- AWARDS
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                AWARDS
              </a>
            </li>-->

            <!-- OTHERS -->
            <!--<li><a href="#contact">Contact</a></li>


            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="#">Action</a></li>
                <li><a href="#">Another action</a></li>
                <li><a href="#">Something else here</a></li>
                <li role="separator" class="divider"></li>
                <li class="dropdown-header">Nav header</li>
                <li><a href="#">Separated link</a></li>
                <li><a href="#">One more separated link</a></li>
              </ul>
            </li>-->

          </ul>
        </div>
      </div>
    </nav>

  </div>
</div>





<div id="myCarousel" class="carousel slide" data-ride="carousel" >


  <div class="carousel-inner" role="listbox">

    <div id="pjback" class="item active">
      <div class="container">

        <div class="bg"></div>

        <div class="carousel-caption" style="
        margin-bottom: 10%;">
          <h1 class="hugetitle">Demonstrate</h1>
        </div>

        <!-- up to next Button-->
        <div class="scroll-down-wrapper">
          <span class="glyphicon glyphicon-chevron-down"></span>
        </div>

      </div>
    </div>


  </div><!-- /.carousel -->
</div>





<!-- Begin Body -->
<div class="mycontainer" id="mycontentcon">
  <div class="row">
    <!--SideBar-->
    <div class="col-md-2">
      <div class="list-group navbar" id="sidebar">
        <ul class="nav mynav" data-spy="affix" data-offset-top="0px">
          <li><a href="#t" class="list-group-item" style="
          height: 40px;
          text-align: center;
          vertical-align: center;
          border-top: 7px solid #f0ad4e !important;
          border-left: 5px solid #f0ad4e !important;
          border-right: 3px solid #f0ad4e !important;
          padding-top: 0px;
          padding-bottom: 0px;
          padding-left: 0 !important;
          padding-right: 0 !important;">
            <div class="progress-container">
              <div class="progress-bar" id="myBar" >
                <span class="progress-value">Demonstrate</span>
              </div>
            </div>
          </a>
          </li>
          <li><a href="#c1" class="list-group-item" style="border-top: 3px solid #f0ad4e !important; ">
            miRNA sensor
          </a>
          </li>
          <li> <a href="#c2" class="list-group-item" contenteditable="false">
            Tet-on swtich
          </a>
          </li>
          <li> <a href="#c3" class="list-group-item" contenteditable="false">
            Carrier and killer：Virus system
          </a>
          </li>
          <li> <a href="#cx" class="list-group-item" contenteditable="false">
            Composite system work
          </a>
          </li>
          <li> <a href="#c4" class="list-group-item" contenteditable="false">
            Prediction and Improvement
          </a>
          </li>
          <li> <a href="#c5" class="list-group-item" contenteditable="false">
            Conclusion
          </a>
          </li>
        </ul>
      </div>
    </div>

    <div class="col-md-9" id="mycontent">
      <h1 id="t" class="">Demonstrate</h1>
      <hr class="col-md-12">
      <p>We wonder if our system works as expected and has the potential to be applied to real-world scenarios. So we demonstrate the feasibility of this project through verification of each independent system, serial effect verification and modeling analysis.</p>

      <div>
        <h1> What had we achieved?</h1>
        <p>1. Sensor: Identified tumor miRNA profile through miRNA sensor</p>
        <p>2. Switch: Verified the response sensitivity of the system switch</p>
        <p>3. Carrier: Reorganized and packaged the virus</p>
        <p>4. Killer: Characterized the efficacy of genes that turn on the virus to kill tumors</p>
        <p>5. The system works normally after being connected in series</p>
        <p>6. Predicated and improved the working condition of oncolytic virus in tumor treatment by modeling</p>
      </div>

      <div>
        <h2 id="c1"> Identify tumor miRNA profile through miRNA sensor</h2>
        <h3>Fitter miRNAs in Colon cancer by modeling </h3>
        <p> We screened miRNA profiles that specifically target colon cancer and examined its specificity in different tissues by modeling, minimizing its off-target effects. Finally, the combination of miRNAs was determined. </p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/c/c0/T--SYSU-CHINA--MVOC.png" style="width:40%; position: relative; margin-left: 30%; margin-right: 30%;">
        <p><strong>Figure 1</strong> Analysis of miRNA volcano maps in colon cancer cells. The left side of the vertical axis indicates the miRNA with low relative expression, and the right side of the vertical axis indicates the miRNA with high relative expression. The larger the absolute value, the larger the differential expression. At the same time, the ordinate indicates the difference significance, and the closer to the horizontal axis, the more significant the difference.</p>
        <p>&nbsp;</p>
        <h3>Differentiation of miRNA profiles using miRNA logic gates </h3>
        <h4>L7Ae K-turn system </h4>
        <p> We engineered the L7Ae protein as an intermediate inhibitor in the miRNA sensor design. (Figure. 2a) By testing we found that it does not have obvious cytotoxicity (Figure.2 c,d) </p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/f/f6/T--SYSU-CHINA--L7Aewhole.png
" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 2</strong>This picture shows the construction and characterization of L7AeO. <strong>a.</strong> According to the 3D structural analysis of the protein, the Thr on the α-helix interacting with the substrate is mutated to Pro by overlap PCR. <strong>b.</strong> 3D structure of L7Ae combined with ligand from PDB    <strong>c.</strong>L7Ae/L7AeO cytotoxicity results microscopy  <strong>d.</strong>Semi-quantitative determination of cytotoxicity by detecting cell reducing ability by MTS staining.</p>
        <p>&nbsp;</p>
        <p> Then we quantitatively and qualitatively characterize the working ability of this system. The result shows that the mutant was weakened as expected but still functions in high concentration. Meanwhile, the 1000ng/ml L7Ae plasmid can depress the translation of mRNA with an efficiency of 84%. (Figure.3a) At the same time, we determined the relationship between the number of K-turn structures on mRNA and the inhibition efficiency. Only two Kt can achieve significant inhibition effect, and the inhibition efficiency of each Kt structure is obtained by data conversion for modeling. (Figure. 3b)</p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/c/c5/T--SYSU-CHINA--DemonsF3.png" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 3</strong> This figure shows the quantitative and qualitative characterization of the L7Ae/L7AeO-Kturn system. <strong>a.</strong> Determination of inhibition efficiency of LA7e/L7AeO and K-turn at different concentrations.(Blue：L7AeO；Black：L7Ae；Green：control plasmid pcDNA). <strong>b.</strong> The effect of different K-turn numbers on L7Ae/L7AeO, only 2 Kt-turn can achieve greater suppression efficiency.</p>
        <p>&nbsp;</p>
        <h4>miRNA target </h4>
        <p>Based on our analysis, we designed three different miRNA targets. We inserted it into the following miRNA detection plasmid (Figure 4a) to verify its independent ability to work and detect the expression of the corresponding miRNA in different HEK293. To verify whether the system causes tandem, we constructed three different plasmid contains 3 miRNA targets independently (miR-592, The detection plasmids of miR-663b and miR885-5p) were tested under different miRNA induction. After the plasmid is transcribed into the cell, the endogenous miRNA binds to the target on the mKate mRNA, blocking the expression of mKate, which cause a change in the value of mKate/EBFP that can be read by flow cytometry. The detection plasmids were tested under different miRNA induction. </p>
        <p>By this method, we can compare the expression of the endogenous corresponding miRNA with the Positive Control which does not carry the miRNA target. At the same time, we transferred miR-592 mimetic, miR-885-5p and miR-663b inhibitors into corresponding cells to detect the regulation of the corresponding miRNA target by miR concentration, thus verifying the rationality and feasibility of single miRNA target design. </p>
        <p>In the flow cytometry analysis (Fig. 4b, 5), we refer to the group with the lower affected group of mKate as the positive group, and the group with the larger influence of mKate as the negative group. We can identify a group of cells that are not obvious in the negative group (hexagonal circled part) in the positive group, and there is a significant difference in the ratio of cells between the positive group and the negative group in the interval of mKate/EGFP<0.4. Thus, we believe that miRNA targets can respond to changes in miRNA concentration.</p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/d/db/T--SYSU-CHINA--MiRMeasurement.jpg" style="width:100%; position: relative; margin-left: 0%; margin-right: 0%;">
        <p><strong>Figure 4</strong> Flow cytometry of miRNA detection plasmids. <strong>a.</strong> The detection plasmid is mainly composed of two fluorescent expression systems. The target of the target miRNA target is inserted into the 3'UTR sequence of mKate. If the miRNA target can work normally, the fluorescence ratio of mKate/EBFP can be changed. <strong>b.</strong> In the results, the positive group has a common cell population, which shows that all three miRNA targets work properly. At the same time, the concentration of endogenous miRNA corresponding to HEK293 can be displayed.</p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/3/3d/T--SYSU-CHINA--mKate.png" style="width:60%; position: relative; margin-left: 20%; margin-right: 20%;">
        <p><strong>Figure 5</strong> Density analysis of positive and unique cell populations revealed that the positive group had a deviation from the negative expression of mKate in the negative group</p>
        <p>&nbsp;</p>

        <h4>miRNA sensor composite system work </h4>
        <p>We constructed a plasmid with a complete miRNA sensor and verified the effects of the entire system at different miRNA concentrations. First, we regulated the concentration of miRNAs in HEK293 through three different miRNA mimics and inhibitors. miRNA-592 mimetic to shut down L7Ae translation, miRNA-663b inhibitor and miR-885-5p inhibitor to block EGFP mRNA degradation. That is to say, when three miRNA mimics are simultaneously transferred, the entire system reaches the maximum working efficiency to output EGFP. The results showed that EGFP was slightly upregulated when either miRNA mimic/inhibitor combination was input, with the combination of miR-663b inhibitor + miR-885-5p inhibitor being the least efficient. This may be due to the fact that the targets of these two miRNAs are on the same mRNA, and the two miRNAs will have a competing effect. In addition, when three miRNA mimics/inhibitors were simultaneously input, we observed a significant difference in fluorescence brightness, which means that our miRNA sensor can work normally only under the expression of a specific miRNA. This undoubtedly verifies its specificity.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/7/7e/T--SYSU-CHINA--CombinationOfDifferentMiRNA.jpg" style="width:60%; position: relative; margin-left: 20%; margin-right: 20%;">
        <p><strong>Figure 6</strong> Different miRNA combinations control the output of miRNA sensors</p>

      </div>

      <div>
        <h2 id="c2"> Verify the response sensitivity of the system switch </h2>
        <h3>Quantitative detection of relationship between Dox concentration and fluorescence induction</h3>
        <p>The result shows that only 270ng/ml DOX can induce half of the gene expression(Figure 7), which means a low dose of DOX can achieve the desired effect without considering the side effect of over-dose.</p>
        <p>&nbsp;</p>
        <img src=" https://2019.igem.org/wiki/images/7/77/T--SYSU-CHINA--rtTA.jpeg" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 7</strong> The effect of Dox concentration on induction efficiency, fitting half of the induction value</p>
        <p>&nbsp;</p>
        <h3> Effect of rtTA3 controlled under different promoters on Tet-on system </h3>
        <p>We also constructed two plasmids which contains the rtTA3 controlled by different promoters. One promoter is pCMV, the other one is hEF1α. The result shows that under the regulation of pCMV, the variance of expression is smaller. While, hEF1α has lower background and higher expression in a high concentration of DOX.(Figure.8) </p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/c/cf/T--SYSU-CHINA--rtTApromoter.gif" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 8</strong> Different promoters control the effect of rtTA3 on Tet-on system</p>
        <p>&nbsp;</p>
      </div>

      <div>
        <h2 id="c3"> Recombined and packaged the virus </h2>
        <p> We used the Adeasy system to insert our circuit into the Ad5 virus missing E1 and E3. First, we constructed the shuttle plasmid pShuttle-AdmiT in vitro to carry our circuit. Then transfer it with pAdeasy into E.coli BJ5183 through electroporation. These two plasmids were recombined in E. coli to obtain the Ad5-type adenovirus genome plasmid carrying our circuit. Figure b is a demonstration of a recombinant plasmid, 1 lane represents a successful recombinant plasmid, lane 2 represents pShuttle, and lane 3 represents a plasmid that has not been successfully recombined.</p>
        <p> We then introduced the plasmid into HEK293 cells and packaged the virus. The image of Day3-8 is shown in the figure. It can be seen that significant CPE appeared on the third day after transfection, and a large number of suspended cells were produced on the eighth day. We harvested the virus suspension on day 8 and stored at -80 ° C. </p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/7/7a/T--SYSU-CHINA--packvir.png" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 9</strong> Results of packaging recombinant adenovirus.  <strong>a)</strong> a plasmid profile of our recombinant adenovirus  <strong>b)</strong> The supercoiled clone in Line 1 is recombinant because it migrate at positions larger than the 10-kb marker and its shuttle vector. However line 3 is not. <strong>c-f)</strong> Use recombinants to transfect the packaging cell line(HEK293), and the CPE showed since 72h.p.t(hours post transfection).</p>
        <p>&nbsp;</p>
        <h3>Determination of virus titer</h3>
        <p>After harvesting the virus, we determined the TCID50 of the recombinant virus. The results showed that the virus titer was 1x10<sup>-3.6</sup>TCID50/ml (Table 1). According to our modeling results, such titers can effectively kill tumors and avoid “escape” effects. </p>
        <p>&nbsp;</p>
        <p><strong>Table 1</strong> The result of TDIC50 assay</p>
        <img src="https://2019.igem.org/wiki/images/b/bf/T--SYSU-CHINA--Demonstrate-virus-titter.png" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p>&nbsp;</p>

        <h3>Killer: Characterized the efficacy of genes that turn on the virus to kill tumors</h3>
        <p>We verified that E1A can restore the virus's self-replication ability by transfection experiments in HeLa cells, and test the killing effect of E1A induced by different concentrations of Dox. This shows that our design in Killer is feasible, we are verifying the impact of the E1B55K, which may be displayed on our posters and presentations. We found that E1A can induce the additive killing of adenovirus in HeLa cells, while the virus lacking E1A cannot increase in cells.</p>
        <p>It can be seen from the cell microscopic (Figure.10) examination that the number of cell deaths increases with increasing E1A concentration. The detection of cell activity on day 4 also supported this result, indicating that our killer gene is effective. (Figure.11) When Dox is 0, E1A also causes apoptosis, which may be caused by pTRE leakage expression. At the same time, since virus infection enhances cell respiration, MTS detects mitochondrial activity, and the virus group data has partial deviation.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/8/8a/T--SYSU-CHINA--E1AtransfectionPicture.jpg " style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 10</strong> Microscopic examination after four days of virus infection</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/b/bc/T--SYSU-CHINA--E1Agraph.jpg " style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 11</strong> Detection of mitochondrial activity by MTS staining for detection of viral effects</p>
        <p>&nbsp;</p>

        <h2 id="cx">The system works normally after being connected in series</h2>
        <p>Based on the above data, we verified the ability of each system to work independently. Now we will test different systems in series to prove that our circuit design is feasible.</p>

        <h3> Tet-on system regulates the initiation of miRNA sensor </h3>
        <p>We placed the miRNA sensor downstream of the pTRE promoter to detect if Tet on could work compatibly with it. miRNA-592 mimics were transported to switch off the miRNA sensor, while miRNA-885-5p inhibitor were transported to switch on the miRNA sensor. Experimental result shows that the two systems are well compatible.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/4/4a/T--SYSU-CHINA--tsys.png" style="width:40%; position: relative; margin-left: 30%; margin-right: 30%;">
        <p><strong>Figure 12</strong> Expression combination of miRNA sensor and Tet on</p>
        <p>&nbsp;</p>
        <h3> Tet-on system regulates the Ad5 to kill cancer cell </h3>
        <p>We co-transfected the adenoviral genome with the E1A gene located downstream of pTRE into HeLa cells. The cells were induced by different concentrations of Dox, and it was found that with the increase of Dox concentration, the cell survival of HeLa cell decreased gradually after 4 days of transfection. This shows that our adenoviral vector system, killer gene and Tet-on system are compatible.</p>
        <!--Data Required-->
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/8/8a/T--SYSU-CHINA--E1AtransfectionPicture.jpg " style="width:50%; position: relative; margin-left: 25%; margin-right: 25%;">
        <p><strong>Figure 13</strong> Tet-on system control virus to kill cancer cell</p>
        <p>&nbsp;</p>
      </div>

      <div>
        <h2 id="c4">Predicated and improved the working condition of oncolytic virus in tumor treatment by modeling</h2>
        <p>Based on the above data, we have modeled and predicted the actual effect of the virus. Such simulations are divided into two processes: intracellular and extracellular.</p>
        <p>In intracellular prediction, we used a randomly distributed model to simulate the workings of the entire circuit after virus infestation in cells. Under the parameters supported by the existing experiments, we found that cancer cells with specific miRNA expression profiles were completely sorted out.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/5/5f/T--SYSU-CHINA--CNC.png" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 14</strong> Different types of cells are distinguished according to EGFP/L7Ae. Cancer cells are clearly sorted out.</p>
        <p>&nbsp;</p>

        <p>In extracellular simulations, we found that existing virus titers can easily kill larger tumors, but small tumors may escape from low titer virus infection(Figure 15). We evaluated this risk and proposed a new mode of administration for multiple doses. Through modeling, we found that with this improvement, viruses with low titer can also apply to tumor treatment in various scenarios (Figure. 16)</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/e/e2/T--SYSU-CHINA--CAF1.png" style="width:100%; position: relative; margin-left: 0%; margin-right: 0%;">
        <p><strong>Figure 15</strong> cancer cell/normal cell ratio <b>(A)</b> and dead cell/normal cell ratio <b>(B)</b> of single-administration method. Solid line: high dose; dashed line: low dose. Red line: big tumors; green line: medium tumors; blue line: small tumors.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/9/9e/T--SYSU-CHINA--CAF3.png" style="width:100%; position: relative; margin-left: 0%; margin-right: 0%;">
        <p><strong>Figure 16</strong> cancer cell/normal cell ratio of re-administration method against small tumors <b>(A)</b> and medium tumors <b>(B)</b>; dead cell/normal cell ratio of re-administration method against small tumors <b>(C)</b> and medium tumors <b>(D)</b>. Color of lines represents different re-administration gap: red, 0 (no re-administration); green, per 24 hours (1 day); blue, per 72 hours (3 days); purple, per 168 hours (7 days). Notably, the scale of y-axis of each graph is different.</p>
        <p>&nbsp;</p>
      </div>

      <div>
        <h2 id="c5">Conclusion</h2>
        <p>In the above description, we first screened out the miRNA combinations through modeling work. In the above description, we first screened the miRNA combination by modeling work, and obtained the miRNA expression profile specifically expressed in colon cancer.</p>
        <p>Then we demonstrated the individual system. For the miRNA sensor system, we verified and tested the parameters and functions of the intermediate system L7Ae-Kturn, single target miR target, and then tested the complete logic gate of the miRNA sensor in HEK293. For the Tet-on system, we tested its induction efficiency and half-fluorescence-induced concentration, while we characterized and compared different promoters in this system. For the adenovirus system, we first verified its recombination ability in E.coli BJ5183 and tested its packaging capacity in HEK293. Based on virus value-added experiments in HeLa cells, we demonstrated that E1A can control viral packaging and demonstrate the ability of the virus to lyse cancer cells under the control of Dox.</p>
        <p>Further, we test different systems in tandem to test their working ability.Further, we tested different working systems in tandem with different systems. The first is the miRNA sensor and Tet-on system, and we demonstrate that the system is compatible with dual regulation of miRNA sensor and Tet-on systems. Secondly, the virus packaging and infection experiments in HeLa cells proved that the packaging and value-added of the virus can be regulated by the Tet-on system through E1A.</p>
      </div>

      <div>
        <h2>Future work</h2>
        <h3>“Plans for the drug administration mode”</h3>
        <p>Both from our human practice work and further investigations into finding a more applicable method in drug delivery, we decided to alter our original drug dosing methods using enema or suppository and recommended a more promising and effective mode, hydrogel, in our drug administration. We compared the advantages and disadvantages of different kinds of drug dosing modes literatures-based, as shown in the table. </p>
        <p>As we can see, although suppository and enema can partly achieve local drug delivery, the administration time and area are too limited. Intestinal stents can meet the needs of long-term administration, but it is easily shifted and needs to be replaced regularly. In light of this, we recommended a recently emerged hydrogel-based drug delivery system, which relies on a physically and chemically crosslinked hydrogel to stably release drugs if it’s activated by particular molecules <sup>[3]</sup>. Although this method is still at experimental stage, its high biocompatibility, convenience and high efficiency make us more convinced that it will eventually be applied clinically. Therefore, we intended to test our system along with this drug dosing method in future verification examinations.</p>
        <p>&nbsp;</p>
        <img src="https://2019.igem.org/wiki/images/1/1f/T--SYSU-CHINA--FutureW.jpg" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 17</strong> Administration concerns</p>

        <h3>“Raising virus titer by modifying the adenovirus vector”</h3>
        <p>Statistics acquired from our modelling work elucidated that higher the virus titer our system produced, more efficient they are in targeting and terminating colon cancer cells. The modelling data also showed that repeated administration can improve the effect in treating cancer. By means of hydrogel-based drug-dosing system, we might realize continuous administrating to the cancer tissues. But its efficiency ought to be further measured by our following work. </p>
        <p>However, according to our experimental data, the recombination efficiency of our adenovirus system was far from perfect. We planned to modify our adenovirus system by replacing the protein fibers on viral capsid from adenovirus type 5 (what we are using now) to adenovirus type 22, and added a layer of or a domain of targeting protein on the viral capsid, by means of molecular cloning. Their expression efficiency differences in eukaryotic cell lines or even tissues could alter be investigated by constructing eukaryotic expression vectors. We intend to test the effect of the combination of the refined adenovirus vector system with our chosen drug-delivery method, for higher the virus titer produced and repeated administration.</p>
        <h2>“Expanding our experiments from human cell lines to a larger extent”</h2>
        <p> We have tested the efficiency of the engineered adenovirus vector system on HEK293, but we still have a long way to go before our project could be safely applied in clinical use. If certain matches of the “hydrogel” administration method worked coordinately with our remodified adenovirus vector, we plan to extend this system to a larger scale, from human cell lines, to colon cancer cell lines, cancer tissues on model animals, and eventually to clinical tests.  Actually, the miRNA profile in our designed system could be artificially reorganized to suit different cancer types, with great possibility of realizing precision medicine at individual level. Additionally, we can further text on multidrug combinations to achieve a better therapeutic effect.</p>
        <h3>References</h3>
        <p>[1] Sepantafar, Mohammadmajid, et al. “Engineered Hydrogels in Cancer Therapy and Diagnosis.” Trends in Biotechnology 35.11(2017).</p>
        <p>[2] Park, Semi , et al. "Benefits of Recurrent Colonic Stent Insertion in a Patient with Advanced Gastric Cancer with Carcinomatosis Causing Colonic Obstruction." Yonsei Medical Journal 50.2(2009).</p>
        <p>[3] English, Max A., et al. "Programmable CRISPR-responsive smart materials." Science 365.6455 (2019): 780-785.</p>
      </div>




      <hr class="col-md-13">
    </div>
  </div>
</div>
</div>



<!-- Back to up Button-->
<div class="scroll-top-wrapper ">
	<span class="scroll-top-inner">
		<span class="glyphicon glyphicon-chevron-up" aria-hidden="true"></span>
	</span>
</div>


<!-- FOOTER -->
<footer id="footer" class="page-footer">
  <!--Footer Links-->
  <div class="container text-center">
    <div class="row1">
      <!--First column-->
      <div class="col-md-3 col-lg-2">
        <h6 class="text-uppercase" style="color: #f9d26b; font-size: 16px; ">
          <strong>Links</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px;">
            FAQ</a>
        </p>
      </div>
      <!--/.First column-->
      <!--Second column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold " style="color: #f9d26b; font-size: 16px; ">
          <strong>Resources</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px; ">
            SYSU</a>
        </p>
      </div>
      <!--/.Second column-->
      <!--Third column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold" style="color: #f9d26b; font-size: 16px; ">
          <strong>Helper</strong>
        </h6>
        <p style="text-align: center;">
          <a href="/education/" style="color: #ffffff; font-size: 14px;">
            Basics (HTML/CSS/JS)</a>
        </p>
      </div>
      <!--/.Third column-->
      <!--Fourth column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold" style="color: #f9d26b; font-size: 16px; ">
          <strong>Company</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px;">
            Our team</a>
        </p>
      </div>
      <!--/.Fourth column-->
    </div>
  </div>
  <!--/.Footer Links-->
  <!-- Copyright-->
  <div class="footer-copyright py-3 text-center">
    <a href="http://2019.igem.org/Team:SYSU-CHINA" style="color: #f9d26b; font-size: 14px;">
      2019 iGEM SYSU-CHINA
    </a>
  </div>
  <!--/.Copyright -->
</footer>



<!-- Bootstrap core JavaScript
================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<!--  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>-->
<!--  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.js"></script>-->
<!--  <script>window.jQuery || document.write('<script src="../carousel/js/jquery.min.js"><\/script>')</script>-->
<!--  <script>window.jQuery || document.write('<script src="../carousel/js/jquery.js"><\/script>')</script>-->

<script src="../carousel/js/jquery.min.js"></script>
<!-- <script src="../carousel/js/jquery.js"></script> -->
<script src="../carousel/js/bootstrap.js"></script>
<!-- Just to make our placeholder images work. Don't actually copy the next line! -->
<script src="../carousel/js/holder.min.js"></script>
</body>
</html>
