<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
  <meta name="description" content="">
  <meta name="author" content="">

  <title>Carousel Template for Bootstrap</title>

  <!-- Bootstrap core CSS -->
  <link href="../carousel/css/bootstrap.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="carousel.css" rel="stylesheet">
</head>


<!-- NAVBAR
================================================== -->
<body>
<div class="navbar-wrapper">
  <div class="container">

    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="https://2019.igem.org/Team:SYSU-CHINA">
            <img src="http://2019.igem.org/wiki/images/1/1f/T--SYSU-CHINA--title0.jpg" alt="logo"
                 style="left:30px; width: 300px; height: 50px; ">
          </a>
        </div>

        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-right .navbar-fixed-top">
            <!-- TEAM -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                TEAM
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="#">Team Members</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Collaborations">Collaborations</a></li>
                <li><a href="#">Awards</a></li>
              </ul>
            </li>

            <!-- PROJECT -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true"
                 aria-expanded="false">
                PROJECT
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Description">Description</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Design">Design</a></li>
                <li><a href="#">Experiments</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Measurement">Measurement</a></li>
                <li><a href="#">Notebook</a></li>
                <li><a href="#">Contribution</a></li>
                <li><a href="#">Results</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Demonstrate">Demonstrate</a></li>
                <li><a href="#">Improve</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Attributions">Attributions</a></li>
              </ul>
            </li>

            <!-- Model -->
            <li class="dropdown">
              <a href="http://2019.igem.org/Team:SYSU-CHINA/Model" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                MODEL
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Description">Description</a></li>
                <li><a href="#">S</a></li>
                <li><a href="#">S</a></li>
                <li><a href="#">R</a></li>
              </ul>
            </li>

            <!-- PARTS -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
                PARTS
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="#">Overview</a></li>
                <li><a href="#">Basic Parts</a></li>
                <li><a href="#">Part Collection</a></li>
              </ul>
            </li>

            <!-- SAFETY -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                SAFETY
              </a>
            </li>

            <!-- HUMAN PRACTICES -->
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false" style="width: 250px;">
                HUMAN PRACTICES
                <span class="caret" style="color: #FFD465;"></span>
              </a>
              <ul class="dropdown-menu" style="min-width:100%;">
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Human_Practices">Human Practices</a></li>
                <li><a href="http://2019.igem.org/Team:SYSU-CHINA/Human_Practices#IHP">Integrated Human Practices</a></li>
              </ul>
            </li>

            <!-- AWARDS
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown"
                 role="button" aria-haspopup="true" aria-expanded="false">
                AWARDS
              </a>
            </li>-->

            <!-- OTHERS -->
            <!--<li><a href="#contact">Contact</a></li>


            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="#">Action</a></li>
                <li><a href="#">Another action</a></li>
                <li><a href="#">Something else here</a></li>
                <li role="separator" class="divider"></li>
                <li class="dropdown-header">Nav header</li>
                <li><a href="#">Separated link</a></li>
                <li><a href="#">One more separated link</a></li>
              </ul>
            </li>-->

          </ul>
        </div>
      </div>
    </nav>

  </div>
</div>





<!-- Carousel
================================================== -->
<div id="myCarousel" class="carousel slide" data-ride="carousel" >


  <div class="carousel-inner" role="listbox">

    <div id="mdback" class="item active">
      <div class="container">

        <div class="bg"></div>

        <div class="carousel-caption" style="
        margin-bottom: 10%;">
          <h1 style="font-size: 64px;">Example headline</h1>
        </div>

        <!-- up to next Button-->
        <div class="scroll-down-wrapper">
          <span class="glyphicon glyphicon-chevron-down""></span>
        </div>

      </div>
    </div>


  </div><!-- /.carousel -->
</div>




<!-- Begin Body -->
<div class="mycontainer" id="mycontentcon">
  <div class="row">
    <!--SideBar-->
    <div class="col-md-2">
      <div class="list-group navbar" id="sidebar">
        <ul class="nav mynav" data-spy="affix" data-offset-top="0px">
          <li><a href="#t" class="list-group-item" style="
          height: 40px;
          text-align: center;
          vertical-align: center;
          border-top: 7px solid #f0ad4e !important;
          border-left: 5px solid #f0ad4e !important;
          border-right: 3px solid #f0ad4e !important;
          padding-top: 0px;
          padding-bottom: 0px;
          padding-left: 0 !important;
          padding-right: 0 !important;">
            <div class="progress-container">
              <div class="progress-bar" id="myBar" >
                <span class="progress-value">Design</span>
              </div>
            </div>
          </a>
          </li>
          <li><a href="#c1" class="list-group-item" style="border-top: 3px solid #f0ad4e !important; ">
            Overview
          </a>
          </li>
          <li> <a href="#c2" class="list-group-item" contenteditable="false">
            Carrier and Killer
          </a>
          </li>
          <li> <a href="#c3" class="list-group-item" contenteditable="false">
            AdEasy System
          </a>
          </li>
          <li> <a href="#c4" class="list-group-item" contenteditable="false">
            Sensorï¼šmiRNA Sensor
          </a>
          </li>
          <li> <a href="#c5" class="list-group-item" contenteditable="false">
            Switch: Tet-on System
          </a>
          </li>
          <li> <a href="#c6" class="list-group-item" contenteditable="false">
            Summary
          </a>
          </li>
        </ul>
      </div>
    </div>

    <div class="col-md-9" id="mycontent">
      <h1 id="t" class="">Design</h1>
      <hr class="col-md-12">


      <div>
        <h2 id="c1" class="">Overview</h2>
        <img src="https://2019.igem.org/wiki/images/7/73/T--SYSU-CHINA--overview.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p>Based on previous years' work, we realized that the design of a successful cancer biotherapy generally consists of the following four parts: Sensor to identify the tumor cell, safety Switch to control the whole system, Carrier or Deliverer to transport drug into the tumor cell, and finally Killer to eliminate cancer cells.</p>
        <img src="https://2019.igem.org/wiki/images/7/73/T--SYSU-CHINA--overview.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p>On this page, we will carefully describe the selection and design of the four basic components in our cancer biotherapy system AdmiT, while showing how modeling results, safety considerations, and expert advice impact on our design. </p>

      </div>

      <div>
        <h2 id="c2" class="">Carrier and Killer</h2>
        <p>In general, a synthetic biology cancer therapy begins with the need to carry the system into cancer cells, ultimately requiring a killer to kill cancer cells. Such vectors are often lentivirus, adeno-associated virus, and the killer gene is generally a gene that controls autophagy and apoptosis. However, such vectors couldnâ€™t self-replicate, requiring constant input from the outside, and each time they need to be transported from the blood into the tumor. As solid tumors are divided into hypoxic and oxygen-rich areas, this kind of therapy may cause a concentration gradient of drug in the tumor, which make their effect on the anoxic zone is minimal.</p>
        <p>To solve this problem, we hope to find a system that does not require external continuous input, and can transfer itself from a cancer cell to surrounding cancer cells, so that this system has the ability to kill solid tumors regardless of aerobic zone and anaerobic zone. </p>
        <p>The oncolytic virus can be used as both a vector and a killer. This engineered virus can only be propagated in cancer cells and released after cell lysis to infect surrounding cancer cells. In this way, the virus can constantly kill surrounding tumor cells, thereby having the potential to ablate solid tumors. It should be noted that the oncolytic virus is a design concept rather than a naturally occurring virus. </p>
        <p>We decided to use oncolytic adenovirus to serve as a carrier and killer.</p>
        <img src="https://2019.igem.org/wiki/images/8/83/T--SYSU-CHINA--oncocomic.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">

        <h3 class="">Which virus should be used as Biological chassis?</h3>
        <p>There are many kinds of oncolytic viruses, for instance, herpes virus, vaccinia virus, measles virus, and adenovirus. Facing so many choices, which one should we choose? </p>
        <p>Some of them, such as M1 virus and coxsackievirus, are natural viruses. Those natural oncolytic viruses generally rely on deleted critical antiviral sites in cancer cells to function. This special mechanism makes it difficult to use as an engineering chassis and limits the scalability of its target.</p>
        <p>How about herpes simplex virus? It has been widely used in the oncolytic virus transformation. But the large genome size of HSV-1 and some double-copies genes make it hard to operate clinically, limiting its engineering transformation potential</p>
        <p>Some oncolytic viruses come from the retrovirus. Considering the ability to integrate foreign genes into the host cell's chromosome, the retrovirus is not suitable as a carrier. We should avoid systematic recombination of viral components into the human genome.</p>
        <p>Here comes Adenovirus. The payload of many oncolytic viruses is some genes encoded tumor-specific promoter and killer in an adenovirus. As a widely-used expression system, the technology of gene modification of adenovirus vector becomes mature and relatively simple. It can infect human cells and has high-level gene expression, which is more stable and without considering the problem that inserted genes will give rise to the elimination of the virus. This is obviously a better choice in terms of security.</p>
        <img src="https://2019.igem.org/wiki/images/0/04/T--SYSU-CHINA--comparison.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p>To create a new oncolytic adenovirus that can target the cancer cells with our sensor and lyase them, the adenovirus vector is the most suitable choice. It can make sure that our circuit can express in the target cancer cells.
          To sum up, we choose adenovirus vectors as our delivery system mainly for the following reasons:</p>
        <p>  â€¢ A broad range of infectivity and high titer </p>
        <p>  â€¢ Infection does not require an actively dividing host cell </p>
        <p>   â€¢ Adenovirus vector is non-insertional </p>
        <p>  â€¢ E1-inserted recombinant adenovirus can replicate even without the packaging cell lines </p>
        <p>The adenovirus vector we used in our project is the pAdEasy-1, whose packaging plasmids are pShuttle and pAdTrack. We could packaged and harvest adenovirus mix in 293T cell lines using this system.</p>

        <h3 class="">What is AdEasy system?</h3>
        <p>Adenoviral vector is widely used for gene expression studies and therapeutic applications. The most common one pAdEasy-1 is generated from human adenovirus serotype 5 with the deletion of the essential gene E1 and nonessential E3 genes. Therefore, the absence of the E1 gene makes the adenoviral vectors become a Replication-defective virus which cannot replicate without packaging cell lines (e.g. HEK293). AdEasy system is constructed based on two vector below. pAdEasy with whole Adenoviral genomeÎ”E1Î”E3ï¼Œwhile pShuttle contains two homogenous arm which allow it could recombine with pAdEasy-1.</p>
        <img src="https://2019.igem.org/wiki/images/7/7d/T--SYSU-CHINA--pAdEasy_Vector.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p>Using the homologous recombination, the gene of interest can be inserted into the vector in E.coli strain BJ5183, then recombinant adenovirus could be packaged and amplified in HEK293. Once infecting the cell, the virus is transported to the nuclear pore complex with the help of cellular microtubules, whereby the adenovirus particle disassembles. The DNA of recombinant adenovirus is subsequently released, which can enter the nucleus via the nuclear pore. Thus, it can use the gene expression system to express the inserted gene.</p>
        <img src="https://2019.igem.org/wiki/images/2/2b/T--SYSU-CHINA--AdEasy_function.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p>Although the adenoviral vectors can express the foreign gene in host cells, it cannot replicate in those cells. The viral DNA eventually disappears. In order for adenovirus to have the selective self-replication ability required by oncolytic viruses, we need to restore the genes that control its lytic cells.</p>

        <h3 class="">Which gene should be covered?</h3>
        <p>Now we have decided to use adenovirus to be our vector, but the deletion of the genome makes that the adenovirus vector cannot autonomously replicate in all cells but packaging cell lines. If we want to build an oncolytic adenovirus, itâ€™s essential to make the recombination virus become self-replicating in cancer cells, which come to the choice of cover genes. </p>
        <p>As mentioned before, the E1 gene is essential for Virus assembly. The E1 area is mainly divided into three sections, E1A, E1B19K and E1B55K. E1A is an early gene necessary for adenovirus replication, whose gene products can activate the necessary expression of certain cell genes and virus genes for adenovirus replication. E1B 55K blocks p53 from inhibiting cell cycling and stops it from inducing apoptosis, creating a suitable environment for the virus to multiply. </p>
        <p>We first tested the feasibility of the system with E1A, and in the following experiments confirmed that E1A is a necessary gene to replenish adenovirus proliferation ability, E1B55K can enhance its ability as a cofactor of E1A, and the removal of E1B17K enhances adenoviral proliferation. Therefore we propose a new version of AdmiT that uses the binding sequence of E1A and E1B55K as an effector gene.</p>
        <img src="https://2019.igem.org/wiki/images/8/81/T--SYSU-CHINA--killer.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">

        <h3 class="">Whether adenovirus is safe for medical use?</h3>
        <p> This issue has been our focus from the beginning, and we have divided it into three main areas: legal regulations, public awareness and medical practice. In response to different aspects, we conducted a questionnaire on the masses under the guidance of professionals, interviewed doctors' opinions on drug administration, and collected relevant regulations in China. Based on these opinions, we designed the future mode of administration. For details, please refer to future work and Human practice.</p>

      </div>

      <div>
        <h2 id="c3" class="">AdEasy System</h2>
        <p>We propose a novel design and construction model mode of replication selective oncolytic viruses. Mathematical model are used to screen specific tumor miRNA combinations in silico. By utilizing the mechanism of action of miRNAs and the nature of specific RBPs, oncolytic viruses designed according to this model can replicate and lyse in tumor cells with specific miRNA expression profiles. The Tet-on system is used to regulate the entire system so that it can be controlled at the same time from both transcription and translation. </p>
        <p>In this experiment, we will validate this model in cell therapy for colon cancer. Importantly, what we present here is a viable cancer treatment model. Based on its principle, it can theoretically be applied to any tumor cell with qualified miRNA profiles. </p>
      </div>

      <div>
        <h2 id="c4" class="">Sensor: miRNA Sensor</h2>
        <p>To achieve the selective killing, we want to find a sensor that can distinguish the wanted and unwanted cellsï¼Œso the task falls on finding the distinguishing molecular markers. In our selective system, only tumor cells are supposed to be killed. However, as there are too many types of tumor cells, it will be too difficult to kill them all with only one device, we decided to choose one or several types of tumor as our model. At the beginning, we want to find tissue-specific promoters but after massive search, we realized that such promoters cannot be universally activated in all kinds of cancer cells. So we then move to think about using suppressor- miRNA. MicroRNAs can regulate gene expression and control many important process including development, differentiation, apoptosis and proliferation. At the same time, miRNA expression profiling has been widely used in diagnosis and prognosis of tumors.</p>
        <p>After screening the expression map of all kinds of miRNA in different tumor and organs and found out that, there is a special miRNA profile expression in colon cancer cells.ï¼ˆTurn to model for detailsï¼‰</p>

        <h3 class="">Which miRNA?</h3>
        <p>For how to choose these miRNAs, we consulted doctors and experts in the study of miRNAs. To select tumor cellï¼Œthey suggested that the miRNA expression profile used to identify needs to meet the following conditionsï¼š</p>
        <p>1. Differentiating between cancerous cells and tumor cells through logic gates</p>
        <p>2. This expression profile cannot appear in other normal tissues</p>
        <p>3. The fluctuation of this expression spectrum should be as small as possible</p>
        <p>We used a modeling approach to screen for miRNA expression profiles that were specifically expressed in colon cancer, and based on this, we constructed the sensory pathway below. The combination contains 3 types of miRNAï¼š</p>
        <p>â€¢ tumor-highï¼šmiR-592ï¼Œwhich high express in colon cell</p>
        <p>â€¢ tumor-lowï¼šmiR-885-5pï¼Œwhich low express even disappear in colon</p>
        <p>â€¢ tissue-specificï¼šmiR-663bï¼Œwhich specific loss in colon-associated cells</p>

        <h3 class="">How miRNA sensor work?</h3>
        <p>miRNA sensor is the most important tool for us to identify tumor cells, and it functions according to the interaction between miRNA and mRNA. Upon partial or complete binding of the miRNA to the mRNA by Watson-Crick pairing, it will mediate the mRNA to stop translation or even degrade. Based on this mechanism, we used the miRNA sensor proposed by Ron in 2015 as a basic component for detecting endogenous miRNAs. Such miRNA sensors are divided into two basic modes of recognition of high-expression miRNAs and low-expression miRNAs, and can further constitute complex logic gate pathways.</p>
        <h4 class="">(1) Switch for low expression miRNA</h4>
        <p>In this case, we hope to output the product when the endogenous miRNA expression is extremely low. Using the mechanism of miRNA action, its pattern and actual design can be shown in the figure below. Inserting the target sequence complementary to the full length of the miRNA into the 5'UTR or 3'UTR of the target mRNA, the high expression of the miRNA binding to the target will degrade the mRNA, while the lowly expressed miRNA will not cause degradation or inhibition of the mRNA. Thus, the target protein is normally output in an environment where the miRNA is lowly expressed, and this can be regarded as a non-gate switch.</p>
        <img src="https://2019.igem.org/wiki/images/3/3d/T--SYSU-CHINA--miR-L_sensor.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">


        <h4 class="">(2) Switch for high expression miRNA</h4>
        <p>Detecting highly expressed miRNA sensors is different from low-expression miRNAs. Since highly expressed miRNAs degrade mRNA with target, we can insert an inhibitory protein L7Ae in the middle, and miRNAs can prevent the expression of inhibitory proteins and form a double inhibitory system. . In the absence of miR-high, L7Ae expression inhibits the expression of mRNA with Kt-turn downstream. In contrast, highly expressed miRNAs mediate inhibition of L7Ae expression, thereby de-suppressing target protein expression.</p>
        <img src="https://2019.igem.org/wiki/images/e/e5/T--SYSU-CHINA--miR-H_sensor.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">


        <h4 class="">(3) Logic gate switches that identify specific expression profiles</h4>
        <p>Based on the above two mode switches, we can construct a logic gate pathway that recognizes a specific miRNA expression profile, as above. Only an environment that satisfies both the miR-HIGH and miR-Low conditions can be output.</p>
        <img src="https://2019.igem.org/wiki/images/1/12/T--SYSU-CHINA--logic_gate.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">


        <h3 class="">How to design a miR Target?</h3>
        <p>The sequence that binds to miRNA and plays a role in miRNA is called miRNA target. In general mammalian cells, it is only about 6 nt. Incomplete pairing causes mRNA to terminate translation. However, complete pairing of miRNA will lead to degradation of mRNA. In the previous miRNA sensor design, full-length paired miRNAs were widely used, which is said to be effective in reducing the occurrence of sponge effects. At the same time, the number of miRNA target sequences on the mRNA and the insertion of specific sequences also affect its inhibition efficiency. Based on the above considerations and doctor's advice, we propose the following design principles:</p>
        <p>1. miRNA target and miRNA full complement complementary pairing</p>
        <p>2. 4x miRNA target is used to enhance inhibition efficiency</p>
        <p>3. specific sequences will be inserted between miRNA targets to enhances their ability to inhibit</p>

        <h3 class="">miRNA target detection and modeling</h3>
        <p>In the subsequent modeling, we found that the experimental results are not consistent with the modeling results. For further consideration, different methods were used to determine the concentration of target miRNAs in the cells, and we found that the actual concentration of miRNA acting on foreign mRNAs is not exactly equal to its expression level, which is due to the competition of endogenous mRNA and sponge effect presence. Referring to a work in 2018, we obtained the free concentration and the Michaelis constant of the target miRNA by the combination of mathematical analysis and experiment, and used these data to establish a mathematical model of the pathway and made a prediction. For specific work, please refer to modeling.</p>

        <h3 class="">Mediate inhibitor system: L7Ae-Kturn</h3>
        <p>L7Ae is a ribosome protein in Archaeoglobus fulgidus.(satio) This protein could bind to the kturn structure on the mRNA and inhibit the downstream translation. In miRNA sensor, it serves as an inhibitor of output mRNA while L7Ae mRNA is under the control of miR-High. Compared with other inhibitory RNP, L7Ae has high inhibition efficiency and short sequence with only 360bp, but it has been reported to be slightly cytotoxic. We further tested the cytotoxicity and inhibition efficiency by constructing mutants. Please refer to result for more details.</p>
        <p>Cite images created with the PDB ID and associated publication, NGL Viewer (AS Rose et al. (2018) NGL viewer: web-based molecular graphics for large complexes. Bioinformatics doi:10.1093/bioinformatics/bty419), and RCSB PDB.</p>
        <img src="https://2019.igem.org/wiki/images/6/61/T--SYSU-CHINA--L7Ae-Kt.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">

      </div>

      <div>
        <h2 id="c5" class="">Switch: Tet-on System</h2>
        <p>The Tet system is a gene-expression regulatory system controlled by exogenous Dox commonly used in synthetic biology. It consists of two parts: the regulatory protein tTA or rtTA and a downstream response element. TetR is a repressor from E. coli, which can block the downstream expression when it binds to a TetO operon. When tetracycline(Tc) or doxycycline(Dox) is absent, gene expression is repressed and when Tc or Dox is introduced to the medium, it leaves the TetO site and let the gene of interest express. The whole tTA regulatory protein is fused by TetR and an activation domain VP16, which will still bind to TetO when dox is absence. However, by alternating four amino acids of tTA, which called rtTA, it binds to tetO operon when Dox is present. When the 7 consecutive tetO sequences are fused with a minimal CMV promoter, we will get a regulatory promote, TRE, that is activated when tTA or rtTA binds to it.</p>

        <h3 class="">Tet-on or tet-off?</h3>
        <p>Basically, there are two kinds of tet control system. The first one, tet-off, is based on tTA, which will activate downstream expression when Tc or Dox is absence. The second one, tet-on, is based on rtTA, which will activate downstream expression when Dox is added.</p>
        <p>Because the virus may have a potential threat, we hope that the virus will not infect the normal human cells and can shut it down at any time. At the same time, considering that the virus will only work briefly during the treatment, we chose the Tet-on system as a short-circuit switch. The Tet-on system activates downstream expression only above a certain blood concentration, and once stopped, it will block downstream expression from a peak drop. So on the one hand it can be used as a quick switch, on the other hand it guarantees that the virus will only start working when it is administered to reduce accidents.</p>
        <img src="https://2019.igem.org/wiki/images/b/b0/T--SYSU-CHINA--Tet-on.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">

        <h3 class="">Which generation?</h3>
        <p>There are three generations of tet-expression systems now. We expect that our virus protein has low level of leakage expression and a high fold increasement. Based on our former work in 2013, we chose Tet-on 3G system with a low leakage expression and high fold increasement.</p>

        <h3 class="">Which promoter? hEF1Î±, pCMV or pUbc</h3>
        <p>To consistently express the protein rtTA of our circuit in a high level, we compared the three promoters: human elongation factor 1 alpha ( hEF-1Î± ), pCMV and pUbC.</p>
        <p>We hope to select a strong promoter that is highly expressed in different cells, and this promoter sequence should be as short as possible due to the capacity limitation of the adenovirus. pCMV is a strong promoter from the virus and is the shortest among the three, however some literature suggests that its expression differs among variable cell lines. hEF1Î± is a human constitutive promoter that is stably expressed in most cells, but it has a sequence length of >1 kb. The promoter of pUbC is lower in intensity than the other two, while can be stably expressed in most cells as well.</p>
        <p>We tested the expression of rtTA under different promoters and the response of the Tet-on system, and finally selected the pCMV promoter considering the application scenario and experimental difficulty.</p>
        <img src="https://2019.igem.org/wiki/images/0/05/T--SYSU-CHINA--rtTA3.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">

      </div>

      <div>
        <h2 id="c6" class="">Summary</h2>
        <p>Up till now, we have introduced all three components. Combining them together, our cricuit finally works like this: </p>
        <img src="http://2019.igem.org/wiki/images/5/52/T--SYSU-CHINA--joint_design.jpg" style="width:90%;">

      </div>




      <hr class="col-md-13">
    </div>
  </div>
</div>
</div>









<!-- Back to up Button-->
<div class="scroll-top-wrapper ">
	<span class="scroll-top-inner">
		<span class="glyphicon glyphicon-chevron-up" aria-hidden="true"></span>
	</span>
</div>

<!-- up to next Button-->
<div class="scroll-down-wrapper">
  <span class="glyphicon glyphicon-chevron-down""></span>
</div>

<!-- FOOTER -->
<footer id="footer" class="page-footer">
  <!--Footer Links-->
  <div class="container text-center">
    <div class="row1">
      <!--First column-->
      <div class="col-md-3 col-lg-2">
        <h6 class="text-uppercase" style="color: #f9d26b; font-size: 16px; ">
          <strong>Links</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px;">
            FAQ</a>
        </p>
      </div>
      <!--/.First column-->
      <!--Second column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold " style="color: #f9d26b; font-size: 16px; ">
          <strong>Resources</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px; ">
            SYSU</a>
        </p>
      </div>
      <!--/.Second column-->
      <!--Third column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold" style="color: #f9d26b; font-size: 16px; ">
          <strong>Helper</strong>
        </h6>
        <p style="text-align: center;">
          <a href="/education/" style="color: #ffffff; font-size: 14px;">
            Basics (HTML/CSS/JS)</a>
        </p>
      </div>
      <!--/.Third column-->
      <!--Fourth column-->
      <div class="col-md-3">
        <h6 class="text-uppercase font-weight-bold" style="color: #f9d26b; font-size: 16px; ">
          <strong>Company</strong>
        </h6>
        <p style="text-align: center;">
          <a href="#" style="color: #ffffff; font-size: 14px;">
            Our team</a>
        </p>
      </div>
      <!--/.Fourth column-->
    </div>
  </div>
  <!--/.Footer Links-->
  <!-- Copyright-->
  <div class="footer-copyright py-3 text-center">
    <a href="http://2019.igem.org/Team:SYSU-CHINA" style="color: #f9d26b; font-size: 14px;">
      2019 iGEM SYSU-CHINA
    </a>
  </div>
  <!--/.Copyright -->
</footer>



<!-- Bootstrap core JavaScript
================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<!--  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>-->
<!--  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.js"></script>-->
<!--  <script>window.jQuery || document.write('<script src="../carousel/js/jquery.min.js"><\/script>')</script>-->
<!--  <script>window.jQuery || document.write('<script src="../carousel/js/jquery.js"><\/script>')</script>-->

<script src="../carousel/js/jquery.min.js"></script>
<!-- <script src="../carousel/js/jquery.js"></script> -->
<script src="../carousel/js/bootstrap.js"></script>
<!-- Just to make our placeholder images work. Don't actually copy the next line! -->
<script src="../carousel/js/holder.min.js"></script>
</body>
</html>
