{{SYSU-CHINA/bootstrap.css}}
{{SYSU-CHINA/carousel.css}}
{{SYSU-CHINA/navbartop}}
{{SYSU-CHINA/backtotopbutton}}
<html>
<body>
<div class="carousel slide" data-ride="carousel" >
  <div class="carousel-inner" role="listbox">

    <div id="pjback" class="item active">
      <div class="container">

        <div class="bg"></div>

        <div class="carousel-caption" style="
        margin-bottom: 10%;">
          <h1 class="hugetitle">Design</h1>
        </div>

        <!-- up to next Button-->
        <div class="scroll-down-wrapper">
          <span class="glyphicon glyphicon-chevron-down"></span>
        </div>

      </div>
    </div>


  </div><!-- /.carousel -->
</div>




<!-- Begin Body -->
<div class="mycontainer" id="mycontentcon">
  <div class="row">
    <!--SideBar-->
    <div class="col-md-2">
      <div class="list-group navbar" id="sidebar">
        <ul class="nav mynav" data-spy="affix" data-offset-top="0px">
          <li><a href="#t" class="list-group-item" style="
          height: 40px;
          text-align: center;
          vertical-align: center;
          border-top: 7px solid #f0ad4e !important;
          border-left: 5px solid #f0ad4e !important;
          border-right: 3px solid #f0ad4e !important;
          padding-top: 0px;
          padding-bottom: 0px;
          padding-left: 0 !important;
          padding-right: 0 !important;">
            <div class="progress-container">
              <div class="progress-bar" id="myBar" >
                <span class="progress-value">Design</span>
              </div>
            </div>
          </a>
          </li>
          <li><a href="#c1" class="list-group-item" style="border-top: 3px solid #f0ad4e !important; ">
            Overview
          </a>
          </li>
          <li> <a href="#c2" class="list-group-item" contenteditable="false">
            Carrier and Killer
          </a>
          </li>
          <li> <a href="#c3" class="list-group-item" contenteditable="false">
            AdEasy System
          </a>
          </li>
          <li> <a href="#c4" class="list-group-item" contenteditable="false">
            Sensor：miRNA Sensor
          </a>
          </li>
          <li> <a href="#c5" class="list-group-item" contenteditable="false">
            Switch: Tet-on System
          </a>
          </li>
          <li> <a href="#c6" class="list-group-item" contenteditable="false">
            Summary
          </a>
          </li>
        </ul>
      </div>
    </div>

    <div class="col-md-9" id="mycontent">
      <h1 id="t" class="">Design</h1>
      <hr class="col-md-12">


      <div>
        <h2 id="c1" class="">Overview</h2>

        <img src="https://2019.igem.org/wiki/images/7/73/T--SYSU-CHINA--overview.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 1. </strong>The whole circit of our design AdmiT</p>
        <p class="pstyle">Based on previous years' work, we realized that the design of a successful cancer biotherapy generally consists of the following four parts: <b>Sensor</b> to identify the tumor cell, safety <b>Switch</b> to control the whole system, <b>Carrier or Deliverer</b> to transport drug into the tumor cell, and finally <b>Killer</b> to eliminate cancer cells.</p>
        <p class="pstyle">On this page, we will describe the selection and design of the four basic components in our cancer biotherapy system <strong>AdmiT</strong>, meanwhile showing how modeling results, safety considerations, and expert advice have an impact on our design. </p>
        <img src="https://2019.igem.org/wiki/images/8/80/T--SYSU-CHINA--biotherapy-model.png" style="width:50%; position: relative; margin-left: 25%; margin-left: 25%;">
        <p><strong>Figure 2. </strong>Four basic parts of a successful biotherapy of cancer</p>
      </div>

      <div>
        <h2 id="c2" class="">Carrier and Killer</h2>
        <p class="pstyle">In previous iGEM project, a cancer biotherapy begins with the need to carry a system into cancer cells, ultimately requiring a killer to kill cancer cells. Such vectors are often lentivirus, adeno-associated virus, and the killer gene is generally a gene that controls autophagy and apoptosis. However, such vectors couldn’t self-replicate, requiring constant input from the external, and each time they need to be transported from the blood into the tumor cells. As solid tumors are divided into hypoxic and oxygen-rich areas, this kind of therapy may generate a concentration gradient of the drug in the tumor, which makes their effect on the anoxic zone minimal.</p>
        <p class="pstyle">To solve this problem, we hope to find a system that does not require continuous input from the external, and can transfer itself from one cancer cell to its surrounding cancer cells, so that this system could have the ability to kill solid tumors regardless of aerobic zone or anaerobic zone. </p>
        <p class="pstyle"><strong>The oncolytic virus</strong> can be used as both a vector and a killer. This engineered virus can only propagate in cancer cells and released after cell lysis to infect surrounding cancer cells. In this way, the virus can constantly kill surrounding tumor cells, thereby having the potential to ablate solid tumors. It is important to note that the oncolytic virus is a design concept rather than a naturally occurring virus. </p>
        <p class="pstyle">We decided to use oncolytic adenovirus to serve as a carrier and killer.</p>
        <img src="https://2019.igem.org/wiki/images/8/83/T--SYSU-CHINA--oncocomic.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 3. </strong>Oncolytic virus can distinguish cancer cells from normal cells and kill specifically</p>

        <h3 class="">Which virus should be used as Biological chassis?</h3>
        <p class="pstyle">There are many kinds of oncolytic viruses, for instance, herpes virus, vaccinia virus, measles virus, and adenovirus. Facing so many choices, which one should we choose? </p>
        <p class="pstyle">Some of them, such as M1 virus and coxsackievirus, are natural viruses. Those natural oncolytic viruses generally rely on deleted critical antiviral sites in cancer cells to function. This special mechanism makes it difficult to use as an engineering chassis and limits the scalability of its target.</p>
        <p class="pstyle">How about herpes simplex virus? It has been widely used in the oncolytic virus transformation. But the large genome size of HSV-1 and some double-copies genes make it hard to operate clinically, limiting its engineering transformation potential</p>
        <p class="pstyle">Some oncolytic viruses come from the retrovirus. Considering the ability to integrate foreign genes into the host cell's chromosome, the retrovirus is not suitable as a carrier. We should avoid systematic recombination of viral components into the human genome.</p>
        <p class="pstyle">Here comes <strong>Adenovirus</strong>. As a widely-used expression system in gene therapy, the technology of gene modification of adenovirus vector has been mature and relatively simple. It can infect human cells and conduct a high level of gene expression<sup>[1,2]</sup>, which is stable without inserting virus genome into human geome. This is obviously a better choice in terms of security.</p>
        <img src="https://2019.igem.org/wiki/images/0/04/T--SYSU-CHINA--comparison.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 4. </strong> Comparison of different virus vectors used as oncolytic virus chassis</p>
        <p class="pstyle">To create a new oncolytic adenovirus that can target the cancer cells with our sensor and lysis systhem, the adenovirus vector is the most suitable choice. It can make sure that our circuit can express in the target cancer cells.</p>
        <p>To sum up, we choose adenovirus vectors as our delivery system mainly for the following reasons:</p>
        <p class="pstyle tab-ls"> <strong>• A broad range of infectivity and high titer </strong></p>
        <p class="pstyle tab-ls">  <b>• Infection does not require an actively dividing host cell </b></p>
        <p class="pstyle tab-ls"> <strong>• Adenovirus vector is non-insertional </strong></p>
        <p class="pstyle tab-ls">  <b>• E1-inserted recombinant adenovirus can replicate even without the packaging cell lines </b></p>
        <p class="pstyle"> The adenovirus vector we used in our project is the <b>pAdEasy-1</b><sup>[3]</sup>, whose packaging plasmids are <b>pShuttle</b> and <b>pAdTrack</b>. We could package and harvest adenovirus mix in 293T cell lines using this system.</p>

        <h3 class="" id="c3">What is AdEasy system?</h3>
        <p class="pstyle">Adenoviral vector is widely used for gene expression studies and therapeutic applications. The most commonly used one pAdEasy-1 is generated from human adenovirus serotype 5 with the deletion of the essential gene E1 and nonessential E3 genes<sup>[4]</sup>. The absence of the E1 gene makes the adenoviral vector a replication-defective virus, which means that it loses the replicative ability unless in packaging cell lines (e.g. HEK293)<sup>[3]</sup>. </p>
        <p class="pstyle">AdEasy system can be constructed based on two plasmids: pAdEasy and pShuttle. The former one contains the whole adenoviral genome except E1 and E3, while the latter contains two homogenous arms, which allows it to recombine with pAdEasy-1<sup>[5]</sup>.</p>
        <img src="https://2019.igem.org/wiki/images/7/7d/T--SYSU-CHINA--pAdEasy_Vector.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 5. </strong> The two plasimds constituting AdEasy system. The left pAdEasy-1 contains the whole genome of adenovirus except E1 and E3(the grey region). The right pShuttle can load the gene of interest and recombine with pAdEasy-1 with homologous arms.</p>
        <p class="pstyle">Using homologous recombination, the gene of interest can be inserted into the vector by E.coli strain BJ5183, and then recombinant adenovirus could be packaged and amplified in HEK293. Once infecting the cell, the virus will soon be transported to the nuclear pore complex with the help of cellular microtubules, whereby, the adenovirus particle disassembles<sup>[6]</sup>. The DNA of recombinant adenovirus is subsequently released, which can enter the nucleus via the nuclear pore. And later, it can manipulate the host gene expression system to express the inserted gene.</p>
        <img src="https://2019.igem.org/wiki/images/2/2b/T--SYSU-CHINA--AdEasy_function.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 6. </strong> Homologous recombination of pAdEasy-1 and pShuttle and virus packaging in 293 cells</p>
        <p class="pstyle">Although the adenoviral vectors can express the foreign gene in host cells, they cannot stably exist and replicate themselves in those cells. To equip adenovirus with self-replication ability, we need to repair the genes that can control the cell lysis ability in our system.</p>

        <h3 class="">Which gene should be covered?</h3>
        <p class="pstyle">Now we have decided to use adenovirus to be our vector, but the deletion of E1 and E3 disables it to autonomously replicate. If we want to build an oncolytic adenovirus, it’s essential to make the recombination virus a self-replicating one in cancer cells, which comes to the choice of cover genes. </p>
        <p class="pstyle">As mentioned before, the E1 gene is essential for virus assembly. The E1 region is mainly divided into three sections: E1A, E1B19K and E1B55K. E1A is an early gene necessary for adenovirus replication, whose gene products can activate the expression of certain cell genes and virus genes necessary for adenovirus replication<sup>[7]</sup>. E1B55K blocks p53 from inhibiting cell cycling and stops it from inducing apoptosis, creating a suitable environment for the virus to multiply<sup>[8]</sup>.</p>
        <p class="pstyle">We will first test the feasibility of the system with E1A, and in the following experiments confirm that E1A is a necessary gene to replenish adenovirus proliferation ability, while E1B55K can enhance its ability, and the removal of E1B19K enhances adenoviral proliferation. Therefore, we propose a new version of AdmiT that uses the binding sequence of E1A and E1B55K as an effector gene.</p>
        <img src="https://2019.igem.org/wiki/images/8/81/T--SYSU-CHINA--killer.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 7. </strong> Two versions of killer gene in our design. Version I is an early gene necessary for adenovirus replication E1A. Version II adds the cofactor of E1A, E1B55K as an attempt to enhance the killing ability.</p>

        <h3 class="">Whether adenovirus is safe for medical use?</h3>
        <p class="pstyle">we has been focused on this issue since the very beginning, and we considered it in three main aspects: legal regulations, public awareness and medical practice. In response to different aspects, we conducted a public-oriented questionnaire under the guidance of professionals, interviewed doctors' opinions on drug administration, and looked through relevant law and regulation in China. Based on these opinions, we designed the future mode of administration. For details, please refer to future work and Human practices.</p>

      </div>
      <div>
        <h2 id="c4" class="">Sensor: miRNA Sensor</h2>
        <h3>Why miRNA profile?</h3>
        <p class="pstyle">To achieve the selective killing, we want to find a sensor that can distinguish the wanted and unwanted cells，so the task falls on finding the distinguishing molecular markers<sup>[9-11]</sup>. In our selective system, only tumor cells are supposed to be killed. However, as there are too many types of tumor cells, it will be too difficult to kill them all with only one device, we decided to choose one or several types of tumor as our model. At the beginning, we want to find tissue-specific promoters. But, after massive search, we realized that such promoters cannot be universally activated in all kinds of cancer cells. So we then move to think about using suppressor- miRNA. MicroRNAs can regulate gene expression and control many important processes including development, differentiation, apoptosis and proliferation. At the same time, miRNA expression profiling has been widely used in diagnosis and prognosis of tumors<sup>[12]</sup>.</p>
        <p class="pstyle">After screening the expression map of all kinds of miRNA in different tumor and organs and found out that, there is a special miRNA profile expression in colon cancer cells<sup>[13]</sup>.（Turn to model for details）</p>

        <h3 class="">Which miRNA?</h3>
        <p class="pstyle">For how to choose these miRNAs, we consulted doctors and experts in the study of miRNAs. To select tumor cell，they suggested that the miRNA expression profile used to identify needs to meet the following conditions：</p>
        <p class="pstyle tab-ls"><b>(1) Differentiating between cancerous cells and tumor cells through logic gates</b></p>
        <p class="pstyle tab-ls"><b>(2) This expression profile cannot appear in other normal tissues</b></p>
        <p class="pstyle tab-ls"><b>(3) The fluctuation of this expression spectrum should be as small as possible</b></p>
        <p class="pstyle">We used a modeling approach to screen for miRNA expression profiles that were specifically expressed in colon cancer, and based on this, we constructed the sensory pathway below. The combination contains 3 types of miRNA：</p>
        <p class="pstyle tab-ls"><b>• tumor-high：miR-592，which high expresses in colon cell</b></p>
        <p class="pstyle tab-ls"><b>• tumor-low：miR-885-5p，which low expresses even disappear in colon</b></p>
        <p class="pstyle tab-ls"><b>• tissue-specific：miR-663b，which specific losses in colon-associated cells</b></p>

        <h3 class="">How miRNA sensor work?</h3>
        <p class="pstyle">miRNA sensor is the most important tool for us to identify tumor cells, and it functions according to the interaction between miRNA and mRNA. Upon partial or complete binding of the miRNA to the mRNA by Watson-Crick pairing, it will mediate the mRNA to stop translation or even degrade. Based on this mechanism, we used the miRNA sensor proposed by Ron in 2015 as a basic component for detecting endogenous miRNAs<sup>[14]</sup>. Such miRNA sensors are divided into two basic modes of recognition of high-expression miRNAs and low-expression miRNAs, and can further constitute complex logic gate pathways<sup>[15]</sup>.</p>
        <h4 class="">(1) Switch for low expression miRNA</h4>
        <p class="pstyle">In this case, we hope to output the products when the endogenous miRNA expression is extremely low. Using the mechanism of miRNA action, its pattern and actual design can be shown in the figure below. Inserting the target sequence complementary to the full length of the miRNA into the 5'UTR or 3'UTR of the target mRNA, the high expression of the miRNA binding to the target will degrade the mRNA, while the lowly expressed miRNA will not cause degradation or inhibition of the mRNA. Thus, the target protein is normally output in an environment where the miRNA is lowly expressed, and this can be regarded as a non-gate switch.</p>
        <img src="https://2019.igem.org/wiki/images/3/3d/T--SYSU-CHINA--miR-L_sensor.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 8. </strong> miRNA sensor response circuit for lowly expressed miRNA and its corresponding logic gate</p>


        <h4 class="">(2) Switch for high expression miRNA</h4>
        <p class="pstyle">Detecting highly expressed miRNA sensors is different from low-expression miRNAs. Since highly expressed miRNAs degrade mRNA with target, we can insert an inhibitory protein L7Ae in the middle<sup>[16]</sup>, and miRNAs can prevent the expression of inhibitory proteins and form a double inhibitory system. In the absence of miR-GIGH, L7Ae inhibits the expression of mRNA with Kt-turn downstream. In contrast, highly expressed miRNAs mediate inhibition of L7Ae expression, thereby de-suppressing target protein expression.</p>
        <img src="https://2019.igem.org/wiki/images/e/e5/T--SYSU-CHINA--miR-H_sensor.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 9. </strong> miRNA sensor response circuit for highly expressed miRNA and its corresponding logic gate</p>



        <h4 class="">(3) Logic gate switches that identify specific expression profiles</h4>
        <img src="https://2019.igem.org/wiki/images/1/12/T--SYSU-CHINA--logic_gate.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 10. </strong> miRNA sensor response circit combining highly and lowly expressed miRNAs and its corresponding logic gate</p>

        <p class="pstyle">Based on the above two mode switches, we can construct a logic gate pathway that recognizes a specific miRNA expression profile, as above. Only an environment that meets both the miR-HIGH and miR-Low conditions can cause output.</p>


        <h3 class="">How to design a miR Target?</h3>
        <p class="pstyle">The sequence that binds to miRNA and plays a role in miRNA is called miRNA target. In general mammalian cells, it is only about 6 nt. Incomplete pairing causes mRNA to terminate translation. However, complete pairing of miRNA will lead to degradation of mRNA. In the previous miRNA sensor design, full-length paired miRNAs were widely used, which is said to be effective in reducing the occurrence of sponge effects. At the same time, the number of miRNA target sequences on the mRNA and the insertion of specific sequences also affect its inhibition efficiency. Based on the above considerations and doctor's advice, we propose the following design principles:</p>
        <p class="pstyle tab-ls"><strong>(1) miRNA target is fully complementary with miRNA</strong></p>
        <p class="pstyle tab-ls"><b>(2) 4x miRNA target is used to enhance inhibition efficiency</b></p>
        <p class="pstyle tab-ls"><b>(3) specific sequences will be inserted between miRNA targets to enhance their ability to inhibit</b></p>

        <h3 class="">miRNA target detection and modeling</h3>
        <p class="pstyle">In the subsequent modeling, we found that the experimental results are not consistent with the modeling results. For further consideration, different methods were used to determine the concentration of target miRNAs in the cells, and we found that the actual concentration of miRNA acting on foreign mRNAs is not exactly equal to its expression level, which is due to the <b>competition of endogenous mRNA</b> and <b>sponge effect</b> presence. Referring to a work in 2018, we obtained the free concentration and the Michaelis constant of the target miRNA by the combination of mathematical analysis and experiment, and used these data to establish a mathematical model of the pathway and made a prediction. For specific work, please refer to modeling.</p>

        <h3 class="">Mediate inhibitor system: L7Ae-Kturn</h3>
        <img src="https://2019.igem.org/wiki/images/6/61/T--SYSU-CHINA--L7Ae-Kt.png" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 11. </strong>  Mediate inhibitor system L7Ae-Kturn to achieve AND gate</p>

        <p class="pstyle">L7Ae is a ribosome protein in Archaeoglobus fulgidus. This protein could bind to the kturn structure on the mRNA and inhibit the downstream translation. In miRNA sensor, it serves as an inhibitor of output mRNA while L7Ae mRNA is under the control of miR-High. Compared with other inhibitory RNP, L7Ae has high inhibition efficiency and short sequence with only 360bp, but it has been reported to be slightly cytotoxic. We further tested the cytotoxicity and inhibition efficiency by constructing mutants. Please refer to result for more details.</p>

        <img src="https://2019.igem.org/wiki/images/a/aa/T--SYSU-CHINA--L7AeStructure.jpg" style="width:80%; position: relative; margin-left: 10%; margin-right: 10%;">
        <p><strong>Figure 12. </strong>  3D structure of L7Ae and Kturn binding</p>
        <p class="pstyle"><small>Cite images created with the PDB ID and associated publication, NGL Viewer (AS Rose et al. (2018) NGL viewer: web-based molecular graphics for large complexes. Bioinformatics doi:10.1093/bioinformatics/bty419), and RCSB PDB.</small></p>
      </div>

      <div>
        <h2 id="c5" class="">Switch: Tet-on System</h2>
        <p class="pstyle">The Tet system is a gene-expression regulatory system controlled by exogenous doxycycline (Dox) commonly used in synthetic biology<sup>[17]</sup>. It consists of two parts: the regulatory protein tTA or rtTA and a downstream response element. TetR is a repressor from E. coli, which can block the downstream expression when it binds to a TetO operon. When tetracycline(Tc) or doxycycline(Dox) is absent, gene expression is repressed and when Tc or Dox is introduced to the medium, it leaves the TetO site and let the gene of interest express. The whole tTA regulatory protein is fused by TetR and an activation domain VP16, which will still bind to TetO when dox is absence. However, by alternating four amino acids of tTA, which called rtTA, it binds to tetO operon when Dox is present. When the 7 consecutive tetO sequences are fused with a minimal CMV promoter, we will get a regulatory promote, TRE, that is activated when tTA or rtTA binds to it.</p>

        <h3 class="">Tet-on or tet-off?</h3>
        <p class="pstyle">Basically, there are two kinds of tetracycline (Tet) control system. The first one, Tet-off, is based on tTA, which will activate downstream expression when Tc or Dox is absence. The second one, Tet-on, is based on rtTA, which will activate downstream expression when Dox is added.</p>
        <p class="pstyle">Because the virus may have a potential threat, we hope that the virus will not infect the normal human cells and can shut it down at any time. At the same time, considering that the virus will only work briefly during the treatment, we chose the Tet-on system as a short-circuit switch. The Tet-on system activates downstream expression only when its level is above a certain blood concentration, and once stop Dox, it will block downstream expression from a peak drop. So on the one hand it can be used as a quick switch, on the other hand it guarantees that the virus will only start working when Dox is administered in order to reduce accidents.</p>
        <img src="https://2019.igem.org/wiki/images/b/b0/T--SYSU-CHINA--Tet-on.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 13. </strong>  The function mechanism of tet-on system </p>

        <h3 class="">Which generation?</h3>
        <p class="pstyle">There are three generations of tet-expression systems now. We expect that our virus protein has low level of leakage expression and a high fold increasement. Based on our former work in 2013, we chose Tet-on 3G system with a low leakage expression and high fold increasement.</p>

        <h3 class="">Which promoter? hEF1α, pCMV or pUbc</h3>
        <p class="pstyle">To consistently express the protein rtTA of our circuit in a high level, we compared the three promoters: human elongation factor 1 alpha ( hEF-1α ), pCMV and pUbC.</p>
        <p class="pstyle">We hope to select a strong promoter that is highly expressed in different cells, and this promoter sequence should be as short as possible due to the capacity limitation of the adenovirus. pCMV is a strong promoter from the virus and is the shortest among the three, however some literature suggests that its expression differs among variable cell lines. hEF1α is a human constitutive promoter that is stably expressed in most cells, but it has a sequence length of >1 kb. The promoter of pUbC is lower in intensity than the other two, while can be stably expressed in most cells as well.</p>
        <p class="pstyle">We tested the expression of rtTA under different promoters and the response of the Tet-on system, and finally selected the pCMV promoter considering the application scenario and experimental difficulty.</p>
        <img src="https://2019.igem.org/wiki/images/0/05/T--SYSU-CHINA--rtTA3.jpg" style="width:80%; position: relative; margin-left: 10%; margin-left: 10%;">
        <p><strong>Figure 14. </strong> Selection of a stably expressing and strong promoter for rtTA3, the regulatory protein in tet-on system </p>

      </div>

      <div>
        <h2 id="c6" class="">Summary</h2>
        <p class="pstyle">Up till now, we have introduced all three components. Combining them together, our cricuit finally works like this: </p>
        <img src="http://2019.igem.org/wiki/images/5/52/T--SYSU-CHINA--joint_design.jpg" style="width:90%;">
        <p><strong>Figure 15. </strong> The function model of our design. Adenovirus invades tumor cells and releases its DNA in the nuclear, and then our circuit express with Dox existing. Only in specific tumor cells whose miRNA profile match the desired design can adenovirus replicate, release and lyse tumor</p>

      </div>



      <div>
        <h3>References</h3>

        <p class="pstyle"><small>[1] Niemann, Julia, Kühnel, Florian. Oncolytic viruses: adenoviruses. Virus Genes. October 2017, Volume 53, Issue 5, pp 700–706.</small></p>
        <p class="pstyle"><small>[2] Russell W C. Adenoviruses: update on structure and function[J]. JOURNAL OF GENERAL VIROLOGY. 2009, 90: 1-20.</small></p>
        <p class="pstyle"><small>[3] He T, Zhou S, Da Costa L T, et al. A simplified system for generating recombinant adenoviruses[J]. Proceedings of the National Academy of Sciences. 1998, 95(5): 2509.</small></p>
        <p class="pstyle"><small>[4] Decaprio J A. How the Rb tumor suppressor structure and function was revealed by the study of Adenovirus and SV40[J]. VIROLOGY. 2009, 384(2SI): 274-284.</small></p>
        <p class="pstyle"><small>[5] Sijing Liu, Weitao Guo, Hui Wang. New progress in adenovirus vector research [J]. Modern Journal of Integrated Traditional Chinese and Western Medicine. 2010, 19(13): 1669-1670.</small></p>
        <p class="pstyle"><small>[6]Meier O, Greber UF (February 2004). "Adenovirus endocytosis". The Journal of Gene Medicine. 6 Suppl 1 (Suppl 1): S152–63.</small></p>

        <p class="pstyle"><small>[7] Berk A J. Functions of adenovirus E1A[J]. Cancer surveys, 1986, 5(2): 367-387.</small></p>

        <p class="pstyle"><small>[8] Martin M E D, Berk A J. Adenovirus E1B 55K represses p53 activation in vitro[J]. Journal of virology, 1998, 72(4): 3146-3154.</small></p>

        <p class="pstyle"><small>[9] Kirwan A,Utratna M,O'dwyer ME,et al. Glycosyl⁃ ation-based serum biomarkers for cancer diagnostics and prognostics[J]. Biomed Res Int,2015,20 (14):4905-4931.</small></p>
        <p class="pstyle"><small>[10] Liloglou T,Bediaga NG,Brown BR,et al. Epigenetic biomarkers in lung cancer[J]. Cancer Lett, 2014,342(2):200-212. </small></p>
        <p class="pstyle"><small>[11] Mabert K,Cojoc M,Peitzsch C,et al. Cancer bio⁃marker discovery:current status and future perspec⁃ tives[J]. Int J Radiat Biol,2014,90(8):659- 677.</small></p>
        <p class="pstyle"><small>[12] Kioomars Saliminejad, Hamid Reza Khorram Khorshid, Shahrzad Soleymani Fard, Seyed Hamidollah Ghaffari. An overview of microRNAs: Biology, functions, therapeutics, and analysis methods. J Cell Physiol. 2018;1–15.</small></p>
        <p class="pstyle"><small>[13] Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, Peck D, Sweet-Cordero A, Ebert BL, Mak RH, Ferrando AA, Downing JR, Jacks T, Horvitz HR, Golub TR. MicroRNA expression profiles classify human cancers. Nature. 435, 834-838.</small></p>
        <p class="pstyle"><small>[14] Zhen Xie, Liliana Wroblewska, Laura Prochazka, Ron Weiss, Yaakov Benenson. Multi-Input RNAi-Based Logic Circuit for Identification of Specific Cancer Cells. SCIENCE VOL 333 2 SEPTEMBER 2011.</small></p>
        <p class="pstyle"><small>[15] Kenji Miki, Kei Endo, ..., Hirohide Saito, Yoshinori Yoshida. Efficient Detection and Purification of Cell Populations Using Synthetic MicroRNA Switches. Cell Stem Cell 16, 699–711, June 4, 2015.</small></p>
        <p class="pstyle"><small>[16] Liliana Wroblewska, Velia Siciliano, Breanna Stillo, Hirohide Saito,Ron Weiss, et al. mammalian synthetic circuits with RNA binding proteins for RNA-only delivery. Nature Biotechnology. Volume 33 Number 8 August 2015.</small> </p>
        <p class="pstyle"><small>[17] Gossen M, Bujard H.Tight control of gene expression in mammalian cells by tetracycline-responsive promoters. Proc Natl Acad Sci USA, 1992,89(12):5547-5551.</small></p>

      </div>


      <hr class="col-md-13">
    </div>
  </div>
</div>
</div>









<!-- Bootstrap core JavaScript
================================================== -->
<!-- Placed at the end of the document so the pages load faster -->


<script type="text/javascript" src="https://2019.igem.org/wiki/index.php?title=Template:SYSU-CHINA/jquery.js&action=raw&ctype=text/javascript"></script>

<script type="text/javascript" src="https://2019.igem.org/wiki/index.php?title=Template:SYSU-CHINA/bootstrap.js&action=raw&ctype=text/javascript"></script>

</body>
</html>

{{Template:SYSU-CHINA/footer}}
